File: applicationProjectExt2/pipelines/Pipeline1/code/src/main/scala/simpledatalabs_27/applicationprojectext2dev/pipeline/pipeline1/graph/Subgraph_1/Reformat_3.scala
package simpledatalabs_27.applicationprojectext2dev.pipeline.pipeline1.graph.Subgraph_1

import io.prophecy.libs._
import simpledatalabs_27.applicationprojectext2dev.functions._
import simpledatalabs_27.applicationprojectext2dev.pipeline.pipeline1.graph.Subgraph_1.config.ConfigStore._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Reformat_3 {
  def apply(spark: SparkSession, in: DataFrame): DataFrame = in
}

File: MigrationPy/pipelines/p2/code/p1/graph/sg2/Reformat_3.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from prophecy.transpiler import call_spark_fcn
from prophecy.transpiler.fixed_file_schema import *
from .config import *
from p1.udfs.UDFs import *

def Reformat_3(spark: SparkSession, in0: DataFrame) -> DataFrame:
    return in0.select(lit(Config.arrayofrec[0].anotherarray[0].var2).alias("first_name"))
